types/postcss-nested/postcss-nested-tests.ts

```diff
@@ -0,0 +1,12 @@
+import * as postcss from 'postcss';
+import * as nested from 'postcss-nested';
```

Comment:
Usually, the import form corresponding to `export = nested` would be `import nested = require('postcss-nested')`.  That form has the advantage of making `nested` callable.